### ISSUE TITLE
review R-CMD-CHECK action

### ISSUE THREAD

**Issue by adokter** [adokter/bioRad#518]
@adokter  has adjusted the R-CMD-check.yaml workflow from vol2birdR in PR https://github.com/adokter/bioRad/pull/517, making edits in the order and changing pak::local_install_dev_deps(upgrade = TRUE) to pak::local_install_deps(upgrade = FALSE), such that the checks also work for OSX.

This is a placeholder for @peterdesmet to review those edits

**Comment by peterdesmet** [comment]
Find my review at https://github.com/adokter/bioRad/pull/517#pullrequestreview-1052046543

**Comment by adokter** [comment]
@peterdesmet thanks. In the old action we had
```
{os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
```
Do you know what the http-user-agent statement does and if we still need it?

**Comment by adokter** [comment]
addressed in #520

**Comment by adokter** [comment]
closing as obsolete
